core/src/main/scala/akka/kafka/scaladsl/Transactional.scala

```diff
@@ -137,7 +138,7 @@ object Transactional {
 
     val flow = Flow
       .fromGraph(
-        new TransactionalProducerStage[K, V, ConsumerMessage.PartitionOffset](settings, settings.transactionIdPrefix + transactionalId)
+        new TransactionalProducerStage[K, V, ConsumerMessage.PartitionOffset](settings, transactionalId)
```

Comment:
I had considered adding it to the above method, but wouldn't it be better to keep it at the underlying one? Maybe you chose the top level since this one is deprecated? I had assumed it might be made private is all?

Reply:
the deprecated one expects the user to "design" their entire transaction id, so we shouldn't mess with it, they can (and do) already have whatever prefix they want